gallery/src/components/NxForm/NxFormPage.tsx

```diff
@@ -292,7 +303,8 @@ const NxFormPage = () =>
                         liveExample={NxFormExample}>
       This example shows an asynchronously loading NxForm. The example is contrived such that the
       form fails to load the first time, but does load (after a brief wait) upon a second attempt.
-      Additionally, the first attempt to submit the form fails, while the retry succeeds.
+      Additionally, the first attempt to submit the form fails, while the retry succeeds. The example also{' '}
+      demonstrates the use of the isRequired flag to remove the "Optional" indicator.
```

Comment:
I also don't understand this change.  `isRequired` is used on some of the form children, but not the form itself.  And it doesn't have to do with an "optional" indicator anymore as on this feature branch we've done away with that in favor of the required indicator (the red asterisk).

Reply:
> I misunderstood thinking the optional indicator would still be used in addition to the required indicator, and thought about highlighting the face that the required indicator was used in the example. However, as you mention that the flag it is not on the form directly but instead on its children, it has been removed in https://github.com/sonatype/sonatype-react-shared-components/commit/dd2b57e1b851c3604298eb633190bb798bc371b1

That commit doesn't affect this line, I still see the problematic verbiage.  I also see it on the `NxFormSelectPage` btw.